setup.py

```diff
@@ -20,6 +19,7 @@
     "pydantic",
     "pytesseract",  # TODO: Test what happens if the host machine does not have tesseract installed
     "requests",
+    "easyocr",
```

Comment:
I'm ok with leaving this as a default, since I think most of its dependencies get installed anyway, but technically this is overkill if someone _does_ have tesseract since they'll never use it. We could theoretically check if `tesseract` exists in `setup.py` and add it dynamically, but that is going a bit far IMO.

Thoughts?

Reply:
Its a really minimal add since biggest dependency is pytorch. We could but it in a separate install group but I dont think its too bad to just include.

Reply:
Okay let's keep it for now then